modules/basics/src/main/java/com/opengamma/strata/basics/location/Country.java

```diff
@@ -32,6 +35,9 @@
   /** Serialization version. */
   private static final long serialVersionUID = 1L;
 
+  private static final PropertiesFile COUNTRY_CODES =
+      PropertiesFile.of(ResourceLocator.ofClasspath(Country.class, "country.properties").getCharSource());
```

Comment:
Use `Suppliers.memoize()` to load this dynamically.
Also, parse it into a `BidiMap` when it is loaded for easier lookup later